weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooksEvaluationComparison.ts

```diff
@@ -855,3 +989,224 @@ function fuzzyMatchScorerName(
   const regex = new RegExp(possibleScorerName.replace(/-/g, '.'));
   return scoreNames.find(name => regex.test(name));
 }
+
+// Determines if an evaluation call is an imperative evaluation
+const isImperative = (evalCall: EvaluationEvaluateCallSchema): boolean => {
```

Comment:
I commented on this before (maybe it got lost in your changes), but this seems very fragile (and incorrect). a normal eval should also have an output with keys